src/lib/data/buyables/buyables.ts

```diff
@@ -653,6 +653,13 @@ const Buyables: Buyable[] = [
 		aliases: ['wool ball', 'ball wool'],
 		gpCost: 300
 	},
+	{
+		name: 'Empty bucket pack',
+		outputItems: {
+			[itemID('Bucket')]: 100
+		},
+		gpCost: 1000
```

Comment:
Slightly too close price to in game GE leaving it vulnerable for a temporary flux over 10 gp and therefor might result in infinit GP. 
 ```suggestion
		gpCost: 10_000
```